public/i18n/en.yaml

```diff
@@ -41,6 +41,10 @@ cluster-overview:
     total-persistentvolumes: 'Total Persistent Volumes'
     persistent-volumes-total-capacity: 'Total Capacity'
     nodes: 'Nodes'
+    modules-overview: 'Modules Overview'
+    modules-ready: 'Ready'
+    modules-error: 'Error'
+    modules-warning: 'Warning'
```

Comment:
we have it in `common.stauses`, probably it will be better to not make additional duplicated state
```
  statuses:
    error: Error
    ready: Ready
    unknown: Unknown
```